packages/command-palette/src/index.tsx

```diff
@@ -91,12 +98,44 @@ export function CommandMenuGroup() {
 		setEmptyListNotice,
 		navigate,
 		currentRoute,
+		currentSiteId,
+		useSites,
 	} = useCommandPaletteContext();
-	const { commands, filterNotice, emptyListNotice } = useCommandPalette();
+	const { commands, filterNotice, emptyListNotice, singleSiteMode } = useCommandPalette();
+	const { __ } = useI18n();
+
+	const sites = useSites();
+	const currentSite = sites.find( ( site: { ID: unknown } ) => site.ID === currentSiteId );
+	const adminUrl =
+		currentSite?.options?.wpcom_admin_interface === 'wp-admin'
+			? 'https://' + currentSite.slug + '/wp-admin'
+			: 'https://wordpress.com/home/' + currentSite?.slug;
+
+	// Should just use the name but need Jetpack change for this to work in wp-admin
+	const siteName = currentSite?.name ?? currentSite?.slug;
 
 	useEffect( () => {
-		setFooterMessage( filterNotice ?? '' );
-	}, [ setFooterMessage, filterNotice ] );
+		if ( ! filterNotice && singleSiteMode ) {
```

Comment:
Should we consider showing the footer message if there are at least 2 sites?

Reply:
Can you just clarify this, because I'm wondering if the naming of `singleSiteMode` is confusing matters. Perhaps `inSiteContext` or `hasSiteSelected` would be clearer.

Do you think we should only display the footer message if a user has multiple sites?

Reply:
I was confused by the name tbh hahaha. It made me think about having a single site. 

> Do you think we should only display the footer message if a user has multiple sites?

I'm not sure, having the message on some occasions and not on others can be confusing. But, having an `All sites` link that goes to a listing with just one site could also be confusing. If users with just one site are more common than users with more sites then I would remove it based on the number of sites. No need to block this PR though, we can figure it out later on, it's not something drastic.

Reply:
> If users with just one site are more common than users with more sites then I would remove it based on the number of sites

Yeah, I agree, and that's the reality: most of our users only have one site.

Said that, we currently show the site selector after selecting a command on a global context (e.g. `/sites`) even if the user only have one site. If we hide the footer, I think we should also hide the site selector for consistency.

Reply:
Improving the "I only have one site" scenario definitely feels like a different issue, I'll improve the variable name here though.